documentation/puppet-api/v3/all_environment_modules.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "All Environment Modules",
+  "description": "An array of information about all the modules in all Puppet code environments",
+  "type":        "array",
+  "properties": ["Environment Modules"],
+  "additionalProperties": false
+}
```

Comment:
Does this work if you were to run it through a tool that does schema validation?